.template/addons/docker/docker-compose.test.yml.tt

```diff
@@ -48,6 +48,7 @@ services:
       - DB_HOST=db
       - CI=$CI
       - TEST_RETRY=$TEST_RETRY
+      - DANGER_GITHUB_API_TOKEN=$TOKEN
```

Comment:
Should we change `$TOKEN` to `$DANGER_GITHUB_API_TOKEN` 🤔  TOKEN sound like really generic

Reply:
Thanks, we will change it :)

Reply:
It seems still as $TOKEN ☁️

Reply:
It's okay now I think.